includes/amp-helper-functions.php

```diff
@@ -14,6 +14,26 @@
 use AmpProject\AmpWP\QueryVar;
 use AmpProject\AmpWP\Services;
 
+/**
+ * Determine whether AMP is enabled on the current site.
+ *
+ * @since 2.1.1
+ * @internal
+ *
+ * @return bool Whether enabled.
+ */
+function amp_is_enabled() {
+	/**
+	 * Filters whether AMP is enabled on the current site.
+	 *
+	 * Useful if the plugin is network activated and you want to turn it off on select sites.
+	 *
+	 * @since 0.2
+	 * @since 2.0 Filter now runs earlier at plugins_loaded (with earliest priority) rather than at the after_setup_theme action.
```

Comment:
```suggestion
	 * @since 2.0 Filter now runs earlier at plugins_loaded (with earliest priority) rather than at the after_setup_theme action.
	 * @param bool $enabled Whether the AMP plugin's functionality should be enabled.
```